### PR TITLE
ionic: add page

### PR DESCRIPTION
**Pull request by jobinlawrance**

### CODE REVIEW

File: pages/common/ionic.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# ionic
+
+> Complete open source SDK to build top quality native and progressive web apps using Angular JS and Apache Cordova.  
+
+- Installation.
```
Comment (agnivade) [inline pages/common/ionic.md]: tldr is a man page alternative. So no need to provide installation steps.

File: pages/common/ionic.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+`ionic start {{project_name}}`  
+
+- Create a new Ionic 2 project.
+
+`ionic start {{project_name}} --v2`
+
+- Create a new project with a local template
+
+`ionic start {{project_name}} {{~/path/to/template}}`
```
Comment (agnivade) [inline pages/common/ionic.md]: {{~/path/to/template}} =>  {{path/to/template}}

File: pages/common/ionic.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+- Create project using template on GitHub
+
+`ionic start {{project_name}} {{username}}/{{repo}}`  
+
+- Create project using template on Codepen
+
+`ionic start {{project_name}} {{codepen_url}}`  
+
+- Viewing the app in a browser
```
Comment (agnivade) [inline pages/common/ionic.md]: "Viewing the app" => "View the app"

File: pages/common/ionic.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+- Create a new Ionic 2 project.
+
+`ionic start {{project_name}} --v2`
+
+- Create a new project with a local template
+
+`ionic start {{project_name}} {{~/path/to/template}}`
+
+- Create project using template on GitHub
```
Comment (agnivade) [inline pages/common/ionic.md]: Create "a" project

File: pages/common/ionic.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+`ionic start {{project_name}} {{username}}/{{repo}}`  
+
+- Create project using template on Codepen
```
Comment (agnivade) [inline pages/common/ionic.md]: Create "a" project

File: pages/common/ionic.md (hunk 4 above)
Comment (waldyrious) [inline pages/common/ionic.md]: In the same vein, "using *a* template".

File: pages/common/ionic.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# ionic
+
+> Complete open source SDK to build top quality native and progressive web apps using Angular JS and Apache Cordova.
```
Comment (waldyrious) [inline pages/common/ionic.md]: We tend to omit adjectives ("complete", "open source", "top quality") in the main description. Stick to describing what it is and what it does :)

Btw, "progressive web app" (PWA) seems to be kind of a specialized term, which is [explained even to ionic users](http://blog.ionic.io/what-is-a-progressive-web-app/). I think it's not generic/informative enough to be included here. Maybe "top quality native and progressive web apps" --> "native-like mobile web apps"?

File: pages/common/ionic.md (hunk 1 above)
Comment (waldyrious) [inline pages/common/ionic.md]: To clarify, it's not about need (beginners would certainly welcome help with installation), but about scope: we have very limited space due to our commitment to provide short pages, and we focus on usage documentation; installation instructions (which often can be quite exhaustive to support multiple platforms, package managers, etc.) can clash with these goals, so we choose not to include them for any of the tools we cover. I hope you can understand :)

**Comment by agnivade** [comment]
@jobinlawrance - any updates on this ?

**Comment by agnivade** [comment]
Hi @jobinlawrance - Please sign the cla so that we can move forward on this PR. I am closing this one after getting no response from you. Feel free to reopen this anytime, and we can resume our work on it !

**Comment by waldyrious** [comment]
@jobinlawrance if you have any objections regarding the CLA, we are also open to hear them :)

**Pull request closed without merging**